File: two-sum/twoSum.js
const twoSum = (array, target) => {
    for(let i = 0; i < array.length; i++){
        for(let j = i + 1; j < array.length; j++){
            if(array[i] + array[j] === target){
                return [i, j];
            };
        };
    };
};

module.exports = { twoSum };

//https://www.codewars.com/kata/52c31f8e6605bcc646000082/train/javascript

File: python-venv/two_sum/kata_function/two_sum.py
def two_sum(array, integer):

    for i in range(0, len(array), 1):
        for j in range(i + 1, len(array), 1):
            if array[i] + array[j] == integer:
                return (i, j)

#https://www.codewars.com/kata/52c31f8e6605bcc646000082/train/python